activesupport/test/core_ext/hash/transform_keys_test.rb

```diff
@@ -21,12 +21,14 @@ class TransformKeysTest < ActiveSupport::TestCase
   test "transform_keys returns an Enumerator if no block is given" do
     original = { a: 'a', b: 'b' }
     enumerator = original.transform_keys
+    assert_nil enumerator.size
```

Comment:
For below case. I believe `transform_keys{!}` can't settlement own size :smiley: 
```ruby
origin = {a: 1, A: 2}
origin.transform_keys(&:downcase) #=> {:a=>2}
```